.ci/Jenkinsfile

```diff
@@ -383,7 +383,15 @@ def generateFunctionalTestStep(Map args = [:]){
                       powershell label: "Check for running elasticsearch", script: "Test-NetConnection ${linuxIp} -Port 9200"
                       powershell label: "Check for running kibana", script: "Test-NetConnection ${linuxIp} -Port 5601"
                       powershell label: "Check for running fleet server", script: "Test-NetConnection ${linuxIp} -Port 8220"
-                      cmd script: "cd e2e\\_suites\\${suite} && go test -timeout 60m -v --godog.tags=\"${tags}\"", label: "Run functional tests for ${platformLabels}:${suite}:${tags}"
+                      cmd script: "mkdir outputs"
+                      try {
+                        dir("e2e/_suites/${suite}") {
+                          cmd script: "go test -timeout 60m -v --godog.format=\"junit:..\\..\\..\\outputs\\TEST-windows-${suite}.xml\" --godog.tags=\"${tags}\"", label: "Run functional tests for ${platformLabels}:${suite}:${tags}"
+                        }
+                      } finally {
+                        junit(allowEmptyResults: true, keepLongStdio: true, testResults: "${BASE_DIR}/outputs/TEST-*.xml")
```

Comment:
We need to collect and archive junit tests here, as this code runs in the Windows worker created ad-hoc. I initially thought the junit step was global to the pipeline, but it's local to the worker.

Reply:
```suggestion
                        junit(allowEmptyResults: true, keepLongStdio: true, testResults: "outputs/TEST-*.xml")
```

Reply:
Detected [here](https://github.com/elastic/e2e-testing/pull/1621/commits/630f4ed7478b9f3da6ccbe1c7d914cc309e3a36f) it before seeing your suggestion